proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,128 @@
+# Cosmos Stargate Hub Upgrade Proposal 2
+
+## by Iqlusion
+
+October 31, 2020
+
+### Current Status: Draft
+
+Blocked on Final Code Hashes from the IBC, Tendermint and SDK teams.
+
+## Key Results
+
+------------
+If passed, this governance proposal would commit the Cosmos Hub to upgrading to halting the `cosmoshub-3` at 06:00 UTC on Nov 19 exporting the state and starting `cosmoshub-4` based on gaia 3.0.
+
+## Context
+
+------------
+
+In proposal 27, Iqlusion proposed a comprehensive process to translate the unprecented surface area of this upgrade. We called this process Stargate.
+
+Iqlusion is pleased to report that the Stargate Process has been successfully executed. We believe the chevrons are unlocked and that Hub can safely step throught the Stargate.
+
+After the success of the community approval of the first Stargate Upgrade proposal, we now follow-up with the second Stargate proposal for the Cosmos Hub that will outline the results of the first proposal and the process for upgrading the Hub.
+
+## The Stargate Plan
+
+------------
+The purpose of the Stargate effort was to ensure that Cosmos can fulfill the vision of an Internet of Blockchains in 2020 while mitigating the risks outlined above. It was also an attempt to create a process for complex future on-chain upgrades on the Cosmos Hub. That included the following steps:
+
+1. Run the Stargate Testnet and use this as an integration testing target with widest possible ecosystem participation to help confirm the readiness of the release software.
+
+2. Engage and support critical partners during the integration process so that partners are able to actively participate in the testing process and provide insights into their success with the upgrade.
+
+3. Dedicate significant resources in terms of full time human resources and documentation efforts to ensure that everyone integrated into the Cosmos Hub can self-certify as Stargate-ready.
+
+4. Report back to the Hub on the success of the integration process.
+
+   - We expect that the primary responsibility of the Cosmos Hub is to assess whether we have mitigated the above risks sufficiently.
+
+   - Provide a written report of the entire Stargate effort to enable Hub governance to make an informed decision.
+
+5. At conclusion, we will indicate why we have confidence that an upgrade won't be disruptive to the Hub’s ecosystem.
+
+## Stargate Plan Results
+
+------------
+
+### 1. Ecoystem Participation
+
+The results of ecosystem participation and engagement are available in our [Stargate Ecosystem Readiness Report](https://github.com/cosmosdevs/stargate/blob/master/ecosystem_readiness.md). We list on this page ecosystem partners such as Validators that provided their validator public keys as well as infrastructure partners that confirmed their integration testing the Stargate testnet tags. Validators, exchanges, and wallets have made up the most responsive of their results completing integration testing of the stargate releases.
```

Comment:
Many of these are still `Integration underway` - I guess that matrix could use an update

Reply:
This is expected to be a living document. We expect the hard deadline will motivate a lot of teams to finish the work.